distributions/otelcol-mackerel/generate.go

```diff
@@ -1,3 +1,3 @@
 package otelcolmackerel
 
-//go:generate go tool builder --config=manifest.yaml --skip-compilation
+//go:generate go tool -modfile=../../tool.mod builder --config=manifest.yaml --skip-compilation
```

Comment:
*go generate* always changes current directory to the package directory it is declared, so this **../../** will always works fine.